lib/Runtime/ByteCode/OpCodes.cpp

```diff
@@ -4,9 +4,6 @@
 //-------------------------------------------------------------------------------------------------------
 #include "RuntimeByteCodePch.h"
 
-// We only have one extended range so maximum number of opcode is only 512 unless we add more extended ranges.
-CompileAssert((uint)Js::OpCode::ByteCodeLast < 512);
```

Comment:
Because it's not true anymore, it's been a while now that we changed extended opcode to support full 16 bits instead of just 9 bits.
We happen to now overflow for javascript, but we've long been past it for asm.js/wasm with simd